### ISSUE TITLE
Setup cloud instances of GTFS Validators

### ISSUE THREAD

**Issue by e-lo** [cal-itp/data-infra#14]
- [GTFS Static](https://github.com/MobilityData/gtfs-validator) - should wait for v2 in late January
- [GTFS Realtime](https://github.com/CUTR-at-USF/gtfs-realtime-validator) - probably ready

cc: @mcplanner

**Comment by machow** [comment]
Alright--I've run the GTFS Static validator a bit (both it's master version, and v1.4 release). Here's some quick impressions.

* It was easy to run the docker image they publish to github!
* The validator takes the name of a file, but then saves the validation report to another file (whose name is tricky to guess). This makes using it hard to use as a service (as is).
* We'll likely need to wrap and modify it's behavior a bit to use it in the pipeline.

I made the code I used while exploring a package, along with a Dockerfile we might use in the pipeline.

https://github.com/cal-itp/gtfs-validator-api
